### ISSUE TITLE
hello, i want to ask the code of run_LNLTC is not container dataset classifier(Ld)?  and the result of finnal PT is the paper's "LNPL w/o Rel"?

### ISSUE THREAD

**Issue by li161** [amurtadha/LNPL#5]

**Comment by amurtadha** [comment]
run_LNLTC includes l_d in lines 656
                loss_rev = criterion_noise(logits_rev, evid)
the result of finale PT is LNPL (ALL), It first train with NN and L_d (i.e., _train_nt), second it refine some labels for the final PT training. 
So the finale PT is trained after refining labels resulting from _train_nt

**Comment by li161** [comment]
this result of LNPL (ALL) is the fanal pt ? I always think the result of "final pt" equal "LNPL w/o Rel",


this GNT is equal the LNPL w/o  l_d?

**Comment by amurtadha** [comment]
What do u mean by GNT? if you mean NT, then yes

**Comment by li161** [comment]
ok, thank you, the accuracy in GNT stage is "LNPL w/o Rel", and the accuracy of epoch from 0 to switch_epoch is queal what in paper?

**Comment by amurtadha** [comment]
The accuracy of NN before switch_epoch is a simply warming the network, it means the boundary is not sharp  enough to separate the noisy and clean .
